src/app/pages/list/inline-actions/inline-actions.component.html

```diff
@@ -0,0 +1,93 @@
+        <mat-card-header></mat-card-header>
+        <mat-card-content>
+            <mat-list class="list-section">
+                <blui-info-list-item
+                    [hidePadding]="true"
+                    class="in"
+                    *ngFor="let item of data; let index = index; let lastIndex = last"
+                    data-cy="list-content"
+                    [divider]="isSmall ? 'full' : lastIndex ? null : 'full'"
+                    (mouseover)="showAction(index, true)"
+                    (mouseleave)="showAction(index, false)"
+                >
+                    <span blui-title>{{ item.title }}</span>
+                    <span blui-subtitle>{{ item.subtitle }}</span>
+                    <span blui-info *ngIf="isSmall && data[index].showTag">
+                        <blui-list-item-tag
+                            style="margin-top: 4px"
+                            [backgroundColor]="colors.red[500]"
+                            label="Active"
+                        ></blui-list-item-tag>
+                    </span>
+                    <div blui-right-content *ngIf="!isSmall; else showMenu">
+                        <div class="inline-actions-buttons" *ngIf="data[index].showActionItems">
+                            <button
+                                data-cy="list-item-delete"
+                                matTooltip="Delete"
+                                mat-icon-button
+                                data-cy="list-delete"
+                                (click)="deleteActions(index)"
+                                style="cursor: pointer;"
+                            >
+                                <mat-icon>delete</mat-icon>
+                            </button>
+                            <button data-cy="list-item-save" matTooltip="Save" mat-icon-button>
+                                <mat-icon>bookmark</mat-icon>
+                            </button>
+                            <button data-cy="list-item-archive" matTooltip="Archive" mat-icon-button>
+                                <mat-icon>archive</mat-icon>
+                            </button>
+                        </div>
+                        <div *ngIf="!isSmall && data[index].showTag; else showInfo">
+                            <blui-list-item-tag
+                                style="margin-right: 8px"
+                                *ngIf="data[index].showTag && !data[index].showActionItems"
+                                [backgroundColor]="colors.red[500]"
+                                label="Active"
+                            ></blui-list-item-tag>
+                        </div>
+                    </div>
+                </blui-info-list-item>
+            </mat-list>
+            <div *ngIf="data.length === 0" class="no-items mat-body-2" data-cy="empty-table">
+                No items found. <span class="reset-table" (click)="loadTable()">Reset table</span>
```

Comment:
Can we make this more semantic by avoiding the use of the word table since this is a list? The design shows "Reset data" instead of "Reset table"